src/BorrowBookUI.java

```diff
@@ -10,7 +10,7 @@
 	private UI_STATE state;
 
 	
-	public BorrowBookUI(BorrowBookControl control) {
+	public borrowBookUi(BorrowBookControl control) {// the name of the method should always start and end with the small letter(BorrowBookUi to borrowBookUi)

```

Comment:
the name of the method should always start and end with the small letter(BorrowBookUi to borrowBookUi)

Reply:
that's good